.circleci/verdaccio/config.yml

```diff
@@ -0,0 +1,27 @@
+storage: ./storage
+auth:
+  htpasswd:
+    file: ./htpasswd
```

Comment:
Couldn't find this file in the commits, did you forget to add it or doesn't it matter?

Reply:
It's created by verdaccio automatically, so it should not be committed.

Reply:
An alternative in case you want to be more transparent https://github.com/verdaccio/monorepo/tree/9.x/plugins/auth-memory plugin might helps, it was made mostly for CI environments.

Reply:
Well, if you have `[$all](publish: $all)` I think even anonymous users can just publish so this might be just decoration.

Reply:
Given this note:
> If you want to use this piece of software, do it at your own risk. This plugin is being used for unit testing.

I'd rather keep the current setup 👀